lib/elixir/lib/inspect/algebra.ex

```diff
@@ -900,7 +906,12 @@ defmodule Inspect.Algebra do
   #
   @typep mode :: :flat | :flat_no_break | :break | :break_no_flat
 
-  @spec fits?(width :: integer(), column :: integer(), break? :: boolean(), entries) :: boolean()
+  @spec fits?(
+          width :: non_neg_integer(),
+          column :: non_neg_integer(),
+          break? :: boolean(),
+          entries
+        ) :: boolean()
         when entries:
                maybe_improper_list({integer(), mode(), t()}, {:tail, boolean(), entries} | [])
```

Comment:
I  am not sure what this interger() is but you could tell me if is should be non_neg_integer() or not.